docs/SRS-Volere/SRS.tex

```diff
@@ -252,22 +252,52 @@ \subsection{Products That Can Be Copied}
 
 \section{New Problems}
 \subsection{Effects on the Current Environment}
-\lips
+\begin{enumerate} [{}1.]
+  \item The application should not create redundant workload for tasks already carried out by existing solutions.
+\end{enumerate}
 \subsection{Effects on the Installed Systems}
-\lips
+\begin{enumerate} [{}1.]
+  \item The application should not affect the system it is being run on other than taking in the required inputs
+and providing the appropriate outputs.
+  \item The application should not affect the systems it is interfacing with other than retrieving
+data, and sending data if necessary.
+\end{enumerate}
```

Comment:
This section is supposed to be about the impact this system will have on existing systems of our Clients (i.e. does it's purpose conflict with that of another system at the City, how does this systems interface with existing systems at the City of Hamilton)

Reply:
Adding on to this, I think that these requirements should get into some detail maybe mention mysds, infor eam. These are rather vague as written.

Reply:
I was going off the work of other teams, and the samples provided. They weren't too specific. I agree that the way I worded these was poor.  I changed the wording and made them more specific.